MatomoTracker.php

```diff
@@ -1697,6 +1698,32 @@ public function setRequestTimeout($timeout)
         return $this;
     }
 	
+    /**
+     * Returns the maximum number of seconds the tracker will spend trying to connect to Matomo.
+     * Defaults to 0 seconds (unlimited).
```

Comment:
It does default to 300, right?

Reply:
yes sorry, fixing it

reference for default value https://curl.se/libcurl/c/CURLOPT_CONNECTTIMEOUT.html